internal/app/go-filecoin/internal/submodule/storage_protocol_submodule.go

```diff
@@ -138,3 +149,19 @@ func (sm *StorageProtocolSubmodule) PieceManager() (piecemanager.PieceManager, e
 	}
 	return sm.pieceManager, nil
 }
+
+func (sm *StorageProtocolSubmodule) DataTransfer() datatransfer.Manager {
+	return sm.dataTransfer
+}
+
+func (sm *StorageProtocolSubmodule) RequestValidator() datatransfer.RequestValidator {
+	return sm.requestValidator
+}
+
+func (sm *StorageProtocolSubmodule) clientEventLogger(event iface.ClientEvent, deal iface.ClientDeal) {
```

Comment:
These should be implemented in the `StorageClientNodeConnector`. Implementing them here adds complexity to the nodes relationship to the storage market.